src/openrct2/peep/peep_data.c

```diff
@@ -1841,7 +1841,7 @@ static const uint8 PeepSpriteImage_9855C4_35_unk04[] = { 13, 0, 1, 2, 2, 2, 2, 3
 static const uint8 PeepSpriteImage_9855C4_36_unk04[] = { 30, 0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 10, 10, 10, 11, 12, 11, 12, 11, 12, 11, 12, 11, 11, 11, 11, 11, 13, 14, 15, 0xFF };
 
 
-static const rct_sprite_image PeepSpriteImage_982888[] = {
+static const rct_sprite_image PeepSpriteImage_Normal[] = {
 	{ 6409, PeepSpriteImage_982888_0_unk04 },
```

Comment:
Yeah I worked out what it all represents. This is all related to the peep actions first action is just none so its all the walking and such. 6409 is the base sprite id for sitting. 0_unk04 is the the amount you increment the base to get to the next frames sprite id.